### ISSUE TITLE
RFC Mention the author(s) and the PR and/issue in changelog.

### ISSUE THREAD

**Issue by adrinjalali** [fairlearn/fairlearn#853]
Our changelog right now doesn't credit the author nor does it mention the PR or the issue corresponding to an entry. Having those makes it easier to track the change, example: https://scikit-learn.org/stable/whats_new/v0.24.html

**Comment by hildeweerts** [comment]
Would this be something we add to the merge commit message in one way or another or would you suggest to add them manually when we create the changelog?

(I'm happy with anything as long as it's clear what's expected from me :D)

**Comment by adrinjalali** [comment]
We write the changelog manually, so it's something we need to make sure to do when we add an item there. Example: https://github.com/scikit-learn/scikit-learn/pull/20165/files#diff-c6c2db6c2fe3fd4325496252937b7a3c08ebdffc7c0546bce69e4508f2ba3a6bR126-R128

**Comment by hildeweerts** [comment]
Ah, I see! So basically you add a new item to the changelog each time new code is merged?

**Comment by romanlutz** [comment]
Looks like it. I like it, but we'll all need to be quite disciplined to get this done. 

I think I also remember @adrinjalali saying that there's one file for ALL changes and another for just the "highlights" (as in bigger changes that will provide new features or breaking changes perhaps). Is that accurate?

Also, this is related to #738 in that our current CHANGES.md is just a markdown file in the repo root, while scikit-learn puts this on the webpage by having it formatted in restructured text.

**Comment by adrinjalali** [comment]
Yep, there's the changelog, and then there's the highlights. For now we don't have much stuff in each release for it to need the highlights. But I think the changelog should be on the website.

**Comment by hildeweerts** [comment]
I agree we should add the changelog on the website! I'm not sure what would be the best place for it given the navbar items in the current design: Get started, Learn, API docs, Contribute, Community, About us

Some examples of other projects:
* pandas has a specific navbar item called *Release notes* in their documentation: https://pandas.pydata.org/docs/whatsnew/index.html (I only just now realize they actually have a separate documentation website generated by sphinx and "main" website that's non-sphinx, which makes the navigation between those a bit weird)
* scikit-learn has a *What's new* subitem of the *more* item: https://scikit-learn.org/stable/whats_new/v0.24.html
* tensorflow does not put them on the website, only in the RELEASE.md on Github
* NumPy adds release notes to the user guide: https://numpy.org/doc/stable/release.html

Tagging @nessamilan @fairlearn/fairlearn-maintainers

**Comment by romanlutz** [comment]
Thanks for the overview of what other projects do! Looks like it's not very standardized 🤣 
Just one thing I want to add: right now we have the "version migration" guide as part of the user guide: https://fairlearn.org/main/user_guide/migrating_versions/index.html
I'm not opposed to moving it elsewhere, though, but I feel like it might make sense to have it in the same place as the changelog. 
[Background: @riedgar-ms initially created this I think for the 0.4.6 -> 0.5 entry, and recently @MeekaElla consolidated it with the existing guide for 0.2 -> 0.3 in #803 ]

**Comment by romanlutz** [comment]
This is also highly related to #738 and basically builds upon it. Whoever sets up the changelog as an ReST file can probably tackle both at the same time.

**Comment by TamaraAtanasoska** [comment]
Addressed in https://github.com/fairlearn/fairlearn/pull/1457, closing!
